internal/compiler/passes/lower_popups.rs

```diff
@@ -150,26 +150,16 @@ fn lower_popup_window(
     });
 }
 
-fn create_coordinate(
-    popup_comp: &Rc<Component>,
-    parent_element: &ElementRc,
-    coord: &str,
-) -> NamedReference {
-    let expression = popup_comp
-        .root_element
-        .borrow_mut()
+fn create_coordinate(popup_root_element: &ElementRc, coord: &str) -> NamedReference {
+    let mut elem = popup_root_element.borrow_mut();
+    let expression = elem
         .bindings
         .remove(coord)
```

Comment:
in fact, you wouldn't need to remove anymore. You can just take a NamedReference of `x` as is, no need to even rename it.

but you may want to adjust the geometry (ElementRc.geometry_props) that it doesn't refer to this property anymore. (IIRC, there is another location of the code where we assign dummy property to it)
Something like https://github.com/slint-ui/slint/blob/6e2013e4ffc0acada2cd4c1120a473c4573836ce/internal/compiler/passes/visible.rs#L66 

This would allow things like this to work:

```
 ...
 p := PopupWindow {
    x: 42px;
    Text { text: p.x / 1px; } // Should display "42", not 0
 }
```